### DISCUSSION TITLE
恒温电水壶2Pro支持一键定温烧水

### DISCUSSION THREAD

**Discussion by grey-337** [XiaoMi/ha_xiaomi_home#484] [Ideas]
小米恒温电水壶2Pro希望能像APP一样支持一键定温和烧水，同时支持定温小时数

**Comment by SusanPhevos** [comment]
请问设备 model 是 [yunmi.kettle.v19](https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:water-heater:0000A02A:viomi-e44:1) 吗？
如果是的话，这个设备没有定义 start-cook 方法，也许要用[这个方法](https://github.com/XiaoMi/ha_xiaomi_home/issues/1524#issuecomment-3583898347)启动烧水。
